monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -19,6 +20,10 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
 
   override protected def servicesHandler(state: CreateZuoraSubscriptionState, context: Context, services: Services) =
     services.zuoraService.subscribe(buildSubscribeRequest(state)).map { response =>
+      state.paymentMethod.`type` match {
```

Comment:
No need for a pattern match here, `if`/`else` would be clearer for something this simple.

Also, why not just ```putCloudWatchMetrics(state.paymentMethod.`type`)```?